eval-flake.sh

```diff
@@ -31,8 +34,9 @@ fi
 
 echo "Evaluating $locked_url..." >&2
 
-if ! (cd $tmp_dir && nix eval --no-allow-import-from-derivation --json "path:.#contents") > "$eval_out" 2> "$flake_dir/eval.stderr"; then
-    echo "Flake $locked_url failed to evaluate." >&2
+if ! nix eval --no-allow-import-from-derivation --min-free 1000000000 --json "path:$(readlink -f $tmp_dir)#contents" > "$eval_out" 2> "$flake_dir/eval.stderr"; then
```

Comment:
I wonder if the min-free needs to be there (or needs to be quite that large), but eh